ui/app/mainui/AppMain.qml

```diff
@@ -1276,4 +1197,22 @@ Item {
             keycardPopup.item.open()
         }
     }
+
+    DropAreaPanel {
```

Comment:
Should it also be possible on public chats?

https://user-images.githubusercontent.com/97019400/194829066-f045fa45-37f1-40d3-819a-350dfc1d180f.mov

Reply:
No, this feature is not supported in public chats, only group chats, 1 on 1s and community channels